pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -479,6 +479,7 @@ func TestMakePodSpec(t *testing.T) {
 		rev  *v1.Revision
 		oc   metrics.ObservabilityConfig
 		dc   *apicfg.Defaults
+		dp   deployment.Config
```

Comment:
ugh, what a pain, guess we have two things that both want to be named `dc`! Tempted to suggest we rename the existing `dc` here to `adc` or even `defaults` since apicfg.Defaults isn't aliased anywhere else (based on a quick search anyway) whereas `deployment.Config` is referenced as `dc` in a [couple](https://github.com/knative/serving/blob/60159d8e8d65afc0dc3d25b04724c2fa90d5e20b/pkg/reconciler/revision/resources/deploy_test.go#L1125) [other](https://github.com/knative/serving/blob/60159d8e8d65afc0dc3d25b04724c2fa90d5e20b/pkg/reconciler/revision/resources/queue_test.go#L91) places (and ac has been used in this package a few times as autoscalerconfig, unfortunately).